### ISSUE TITLE
Convert HTML <img> to sphinx AST

### ISSUE THREAD

**Issue by chrisjsewell** [executablebooks/MyST-Parser#208]
This would have the benefit that the size of images could be controlled in  way where the image is still viewable in places like here in GitHub.
Whilst converting it to AST would mean that it is handled consistently in Sphinx (e.g. it would be output correctly in TeX/PDFs)

e.g.

```
<img src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/socialmedia/apple/237/smiling-face-with-smiling-eyes_1f60a.png" alt="emoji" width="200px">
```

<img src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/socialmedia/apple/237/smiling-face-with-smiling-eyes_1f60a.png" alt="emoji" width="200px">

as opposed to

```
![emoji](https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/socialmedia/apple/237/smiling-face-with-smiling-eyes_1f60a.png)
```

![emoji](https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/socialmedia/apple/237/smiling-face-with-smiling-eyes_1f60a.png)

or

````
```{image} https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/socialmedia/apple/237/smiling-face-with-smiling-eyes_1f60a.png
:width: 200px
```
````

```{image} https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/socialmedia/apple/237/smiling-face-with-smiling-eyes_1f60a.png
:width: 200px
```

**Comment by chrisjsewell** [comment]
HTML attributes are listed here: HTML attributes are listed here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

One could also consider converting:

```
 <figure>
  <img src="pic_trulli.jpg" alt="Trulli" width="200px">
  <figcaption>Fig.1 - Trulli, Puglia, Italy.</figcaption>
</figure> 
```

This is a little more involved though and the caption could not be Markdown.
Something like this would probably be better

````
:::{figure}
<img src="pic_trulli.jpg" alt="Trulli" width="200px>
This **is** the caption
:::
````

**Comment by chrisjsewell** [comment]
closed by #210
